tests/test_urldispatch.py

```diff
@@ -1054,14 +1150,14 @@ def test_frozen_router_subapp(app, loop):
     subapp = web.Application()
     subapp.freeze()
     with pytest.raises(RuntimeError):
-        app.add_subapp('/', subapp)
+        app.add_subapp('/pre', subapp)
```

Comment:
Why the test has changed?

Reply:
Empty prefix is ​​not supported, and now the prefix checks first then subapp

Reply:
good